WORKSPACE

```diff
@@ -976,3 +976,10 @@ java_import(
     strip_prefix = "canton-0.15.0",
     urls = ["https://www.canton.io/releases/canton-0.15.0.tar.gz"],
 )
+
+http_file(
```

Comment:
You could use `http_jar` but since you still need the `java_binary` it doesn’t buy you that much.